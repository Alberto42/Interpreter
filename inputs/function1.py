y = 0
def func(x) {
    y=1
}

func(3)