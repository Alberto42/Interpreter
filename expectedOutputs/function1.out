Ident "y" = VInt 1


